src/components/PopupConfirmDelete.js

```diff
@@ -0,0 +1,34 @@
+import Popup from "./Popup";
+
+export default class PopupConfirmDelete extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+    this._closeConfirmForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupElement.querySelector(
+      ".modal__button-delete"
+    );
+    this._submitButtonText = this._submitButton.textContent;
+  }
+
+  setConfirmDelete(func) {
+    this._handleFormSubmit = func;
+  }
+
+  setDeleteText(isDeleting) {
+    if (isDeleting) {
+      this._submitButtonText = "Deleting...";
```

Comment:
You need to use `this._submitButton.textContent` for setting this text while loading


```js

      this._submitButton.textContent = "Deleting...";

```